java/ql/src/semmle/code/java/dataflow/internal/ContainerFlow.qll

```diff
@@ -168,13 +203,44 @@ private predicate argToQualifierStep(Expr tracked, Expr sink) {
   )
 }
 
+/** Access to a method that passes taint from an argument. */
+private predicate argToMethodStep(Expr tracked, MethodAccess sink) {
+  exists(Method m, int i |
+    m = sink.(MethodAccess).getMethod() and
+    taintPreservingArgumentToMethod(m, i) and
+    tracked = sink.(MethodAccess).getArgument(i)
```

Comment:
```suggestion
    m = sink.getMethod() and
    taintPreservingArgumentToMethod(m, i) and
    tracked = sink.getArgument(i)
```

Reply:
Applied suggestion and made the same change to `TaintTrackingUtil` as well.